examples/cucumber/.sauce/config.yml

```
@@ -7,11 +7,10 @@ sauce:
   region: us-west-1
   concurrency: 2 # Controls how many suites are executed at the same time.
   metadata:
-    name: saucectl cypress cucumber example
     tags:
       - cucumber
       - example repo
-    build: Github Run $GITHUB_RUN_ID
+    build: saucectl cypress cucumber example
```

That would at least separate out different pipeline builds from each other:
```suggestion
    build: saucectl cypress cucumber example $GITHUB_RUN_ID
```

PS: Not sure if that's too long now 🤔